src/components/ShoppingListItem/ShoppingListItem.js

```diff
@@ -17,19 +19,27 @@ const ShoppingListItem = ({ listId, itemId, item, checkAsPurchased }) => {
   return (
     <li className="shopping-list__item item">
       <input
-        id={`item-${itemId}`}
-        value={itemId}
+        id={`item-${item.id}`}
+        value={item.id}
         type="checkbox"
         disabled={recentlyPurchased}
         checked={recentlyPurchased}
-        className={`checkbox shopping-list__checkbox ${recentlyPurchased} ? 'checkbox_recently-purchased' : '' `}
-        onChange={() => checkAsPurchased(itemId, item)}
+        className={`checkbox item__checkbox ${
+          recentlyPurchased ? 'checkbox_recently-purchased' : ''
+        } item__checkbox_${item.status}`}
+        onChange={() => checkAsPurchased(item)}
       />
       <label
-        className="label label_check-radio shopping-list__label"
-        htmlFor={`item-${itemId}`}
+        className={`label label_check-radio item__label item__label_${item.status}`}
+        htmlFor={`item-${item.id}`}
       >
         {item.itemName}
+        <span className="visually-hidden">
+          {item.status === 'soon' && ' Need to buy soon'}
+          {item.status === 'kind-of-soon' && ' Need to buy kind of soon'}
+          {item.status === 'not-soon' && " Don't need to buy soon"}
+          {item.status === 'inactive' && ' Inactive'}
+        </span>
```

Comment:
It might be helpful to put a comment that summarizes your note in the PR for what this span is doing.

It's really nice that the screenreader reads "Need to buy soon", "Need to buy kind of soon", etc. However, what is the `soon`, `kind-of-soon` at the beginning of each `&&` statement used for?

Reply:
We thought it made sense to set a property item.status back in the ShoppingList component, since we already needed to figure out which items were inactive there in order to sort. We decided to use the strings 'soon', 'kind-of-soon', 'not-soon' and 'inactive' as the 4 possible values for item.status, and these are returned by the getItemStatus function (lines 128-134 of ShoppingList). The thinking was we could reuse these strings as CSS class modifiers after we passed the item object (with its new status property) to ShoppingListItem. Since we also need to print fuller labels for the benefit of human ears, here we are just checking to see which short little string value is stored in item.status so we know which corresponding full label to print.

Reply:
We can definitely add a short note here and the other places you mentioned @connietran-dev